src/App.js

```diff
@@ -1,7 +1,37 @@
-import React from 'react'
+import React from 'react';
 
-export default function App() {
+const todoList = [
+    {
+    title: 'To finish assignment',
+    objectID: 0,
+    },
+    {
+    title: 'Read the book Road to React',
+    objectID: 1,
+    },
+    {
+    title: 'Watch Youtube videos by me',
+    objectID: 2,
```

Comment:
Please use `id` or `todoId` instead of `objectId`. This is not technically wrong but is verbose and without semantic meaning. If you were to introduce other features(for example, adding category tags to the todos) that included an array of objects, this approach will become very confusing. In cases where you need to disambiguate `id`'s of different type of objects, you can lengthen it to `todoId` and you could use `categoryId` for that hypothetical category tags feature.

Reply:
good to know , thank you